.golangci.yml

```diff
@@ -41,6 +41,10 @@ linters-settings:
     disabled-checks:
       - wrapperFunc
       - dupImport # https://github.com/go-critic/go-critic/issues/845
+  godox:
```

Comment:
please, remove it: these keywords are already enabled by default

Reply:
Removed.